game/hud/src/components/GameMenu/index.tsx

```diff
@@ -17,59 +17,120 @@ export interface GameMenuStyle {
   closeButton: React.CSSProperties;
 }
 
-const Container = styled('div')`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  pointer-events: all;
-  width: 300px;
-  height: 200px;
-  background-color: rgba(0, 0, 0, 0.5);
-  border: 1px solid ${utils.lightenColor('#202020', 30)};
+const OuterContainer = styled('div')`
+  position: relative;
 `;
 
-const Header = styled('div')`
-  padding: 5px 0;
-  text-align: center;
-  background-color: #202020;
+const Container = styled('div')` {
+  position: relative;
+  pointer-events: all;
+  width: 320px;
+  height: 152px;
+  padding: 0px;
+  margin:0 auto;
+  background-color: gray;
   color: white;
-  border-bottom: 1px solid ${utils.lightenColor('#202020', 30)};
+  background: url(images/gamemenu/gamemenu-bg-grey.png) no-repeat;
+  z-index: 1;
+  border: 1px solid #6e6c6c;
+  box-shadow: 0 0 30px 0 #000;
 `;
 
-const Body = styled('div')`
-  padding: 5px;
-  flex: 1;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+const MenuTitle = styled('div')` {
+  text-align: center;
+  background: url(images/gamemenu/gamemenu-top-title.png) center top no-repeat;
+  margin: 0 auto -9px auto;
+  position: relative;
+  z-index: 89;
+  width: 319px;
+  height: 23px;
+  h6 {
+    color: rgb(132,132,132);
+    font-size: 9px;
+    text-transform: uppercase;
+    padding: 8px 0 0 0;
+    margin: 0 0 0 0;
+    letter-spacing: 2px;
+    font-family: 'Caudex', serif;
+  }
 `;
 
-const Footer = styled('div')`
-  display: flex;
-  justify-content: center;
-  padding: 5px 0;
-  background-color: #202020;
-  color: white;
-  border-top: 1px solid ${utils.lightenColor('#202020', 30)}
+const MenuCorner = styled('div')` {
+  position: absolute;
+  min-width: 320px;
```

Comment:
Any reason you set absolute width and heights for these guys? Could just use 100% for the dimensions

Reply:
You don't need to update this, I can do on local comp if there's no problems